package.json

```diff
@@ -6,13 +6,19 @@
     "start": "expo start",
     "android": "expo start --android",
     "ios": "expo start --ios",
-    "eject": "expo eject"
+    "eject": "expo eject",
+    "clear-metro-cache": "rm -rf /tmp/metro-bundler-cache-*",
+    "clear-haste-cache": "rm -rf /tmp/haste-map-react-native-packager-*"
   },
   "dependencies": {
+    "bootstrap": "^4.1.3",
     "expo": "^30.0.1",
+    "native-base": "^2.8.1",
     "prop-types": "^15.6.2",
-    "react": "16.3.1",
+    "react": "^16.3.1",
+    "react-dom": "^16.6.1",
     "react-native": "https://github.com/expo/react-native/archive/sdk-30.0.0.tar.gz",
```

Comment:
It seems that some of these added dependencies are not being used, are they?

Reply:
They are dependencies for the libraries I installed

Reply:
Then they should not be in our package.json. 
Namely, `boostrap` `react` `react-dom`.

Reply:
Ok cool will change that

Reply:
Not seeing the bug btw. Can u send me your number or email I'll send the tunnel link to you?

Reply:
Hey so react is needed because I do import it in ChooseCityScreen and it crashes otherwise. Removed react-dom and bootstrap. Good catch. Updated otherwise.